features/allure.feature

```diff
@@ -2,7 +2,7 @@ Feature: To search allure reports in google
 
     @AllureScenario
     Scenario: Allure Reports Google
-      Given I am on google page
-      When I type "allure reports"
-      Then I click search button "input[value='Google Search']"
+      Given I am on google page with title "Google"
+      When I type "allure reports" on search textbox "#lst-ib"
```

Comment:
Any reason we are passing page objects in feature files? Actually style guide suggests to keep the page objects in a separate object/class. Feature files are strictly for the users to read and understand about the tests. They would not want to see page objects there! Can you please change it?

Reply:
I tried to use this pattern without knowing the style guide, using it in practice and together with his advice, I saw that it was not legal, thanks a lot for the tip!